sale_timesheet_line_exclude/models/account_analytic_line.py

```diff
@@ -27,13 +27,13 @@ def _constrains_exclude_from_sale_order(self):
                     )
                 )
 
+    # pylint: disable=missing-return
```

Comment:
It's not needed to remove this and it's correct to always return the super value (even although Odoo framework is not doing anything with this).